### DISCUSSION TITLE
Howdy!

### DISCUSSION THREAD

**Discussion by nerrixde** [benawad/dogehouse#173] [General]
What a nice chat!
While I might be the first in this forum, I have something useful to contribute:

Which hosting providers do you all like / use?
_I personally enjoy Digitalocean_

**Comment by RazerMoon** [comment]
I don't ever run anything that requires a lot of resources, so I usually just go for the cheapest deal I can find.
Lately I've been using https://www.hetzner.com/, they give you 1 vCPU 2GB RAM 20GB SSD 20TB traffic for 3.01 euro / month. Their servers are based in Germany and I'm pretty close to there which is nice.
